SocialCareCaseViewerApi.Tests/V1/Data/MongoDB/UpdateMosaicIdFormDataCollection.cs

```diff
@@ -0,0 +1,156 @@
+        ///  update: "form_data",
+        ///  updates:
+        ///      [
+        ///          {
+        ///              q: { MosaicId: "334445566", FormNameOverall: { $not: /^API_/ } }, u: { $set: { MosaicId: "77889900" } }, multi: true
+        ///          }
+        ///      ]
+        /// });
+        /// </howto>      
+
+
+        [SetUp]
+        public void SetUp()
+        {
+            string mongoConnectionString = Environment.GetEnvironmentVariable("SCCV_MONGO_CONN_STRING") ??
+                                           Environment.GetEnvironmentVariable("MONGO_CONN_STRING") ??
+                                           @"mongodb://localhost:1433/";
+
+            string databaseName = Environment.GetEnvironmentVariable("SCCV_MONGO_DB_NAME") ?? "social_care_db_name";
+
+            var mongoClient = new MongoClient(new MongoUrl(mongoConnectionString));
+            _mongoDatabase = mongoClient.GetDatabase(databaseName);
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            _mongoDatabase?.DropCollection(MainFormCollection);
+        }
+
+        [Test]
+        public void UpdatesMosaicIdToNewMasterPersonIdForAllMatchingRecords()
+        {
+            string mosaicId = "334445566";
+            string masterPersonId = "77889900";
+
+            var casenoteOne = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicId).Create();
+            var casenoteTwo = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicId).Create();
+            var casenoteThree = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicId).Create();
+
+            var collection = _mongoDatabase?.GetCollection<MongoDBTestObject>(MainFormCollection);
+
+            collection?.InsertOne(casenoteOne);
+            collection?.InsertOne(casenoteTwo);
+            collection?.InsertOne(casenoteThree);
+
+            var result = _mongoDatabase?.RunCommand(_dbCommandUnderTest);
+
+            var filterByNewId = Builders<MongoDBTestObject>.Filter.Eq("MosaicId", masterPersonId);
+            var recordsWithNewId = collection.Find(filterByNewId).ToList();
+            recordsWithNewId.Count.Should().Be(3);
+
+            var filterByOldId = Builders<MongoDBTestObject>.Filter.Eq("MosaicId", mosaicId);
+            var recordsWithOldId = collection.Find(filterByOldId).ToList();
+            recordsWithOldId.Count.Should().Be(0);
+        }
+
+        [Test]
+        public void DoesNotUpdateOtherThanMatchingRecords()
+        {
+            string mosaicId = "334445566";
+            string mosaicIdTwo = "556677";
+            string masterPersonId = "77889900";
+
+            var casenoteOne = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicId).Create();
+            var casenoteTwo = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicId).Create();
+            var casenoteThree = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicIdTwo).Create();
```

Comment:
```suggestion
            var casenoteOneForMosaicId = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicId).Create();
            var casenoteTwoForMosaicId = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicId).Create();
            var casenoteThreeForMosaicIdTwo = _fixture.Build<MongoDBTestObject>().With(x => x.MosaicId, mosaicIdTwo).Create();
```

Or something like that to make it clear that the third case note is for another person.

Reply:
@chubberlisk I've renamed them. That's better